src/iothread.c

```diff
@@ -195,7 +195,7 @@ static int PausedIOThreads[IO_THREADS_MAX_NUM] = {0};
 
 /* Pause the specific range of io threads, and wait for them to be paused. */
 void pauseIOThreadsRange(int start, int end) {
-    if (server.io_threads_num <= 1) return;
+    if (server.io_threads_num <= 1 || !server.io_threads_active) return;
```

Comment:
Can we use `server.io_threads_active` directly to determine whether `io-thread` is started without `io_threads_num`.

Reply:
yes, good point